apps/system/locales/system.en-US.properties

```diff
@@ -647,11 +647,11 @@ pinning-unpin-type=Unpin "{{type}}"
 
 # LOCALIZATION NOTE (pinning-pin-page): This string is used as
 # panel title. "Pin" indicates the action to pin a page.
-pinning-pin-page=Pin Page
+pinning-pin-page=Pin page
 
 # LOCALIZATION NOTE (pinning-pin-site): This string is used as
 # panel title. "Pin" indicates the action to pin a site.
-pinning-pin-site=Pin Website
+pinning-pin-site=Pin site
```

Comment:
The spec says 'Website', isn't it? https://mozilla.app.box.com/s/a25ot7ruu5u0mvxjirhrtu8d3rt5r107